AUTHORS.md

```diff
@@ -0,0 +1,231 @@
+# Authors
+
+This list is sorted by the number of commits per contributor in descending order.
```

Comment:
s/This list is sorted by the number of commits per contributor in descending order/The list is auto-generated by xxx scripts, please do not edit. The list is sorted by the number of commits per contributor in descending order/